### DISCUSSION TITLE
How to use breakdown with bar graph ?

### DISCUSSION THREAD

**Discussion by HiFiPhile** [openobserve/openobserve#4661] [Q&A]
Hi, I'm using openobserve to collect sensor data with format:
```
{
_timestamp:1727366179130577,
data:0006000B,
dstid:00000002,
funcno:0000,
length:4,
mfrid:0000,
rorg:4BS,
rssi:61,
srcid:00000001,
status:00,
timestamp:2024-09-26T17:56:19.128000,
type:Inbound
}
```

With stacked graph `histogram(_timestamp)` as X axis, `count(data) as Y axis` and `srcid` as breakdown I can get a nice graph:
![image](https://github.com/user-attachments/assets/19f2a17a-d0d8-456c-92ef-40f460704f85)

But with bar graph the bars are not breaked down by `srcid` into multiples bars, on the X axis I can see different `srcid` but all in the same bar:
![image](https://github.com/user-attachments/assets/aae494d6-d0d3-43b8-a16b-d1d3a112d996)

**Comment by ktx-kirtan** [comment]
Hi, This would make more sense when you don't have a time-series on the x-axis.

For example, If you want a bar chart on body_method distributed by the body_status. see the example below:

![image](https://github.com/user-attachments/assets/4efd11be-dc57-4aeb-9283-d4f804df7e4c)

**Reply by HiFiPhile** [reply]
Thank you for your reply, what I want is the same data of stacked graph but represented differently by breaking into multiple bars  instead of stack them together.

Is it possible to have something like this ?
![image](https://github.com/user-attachments/assets/d9a68d16-243b-4a5b-8af8-54f21776d985)
